### PR TITLE
Fix: replace ldaps:// by ssl://

### PR DESCRIPTION
**Pull request by hregis**

### CODE REVIEW

File: htdocs/core/class/ldap.class.php [hunk 1]
```diff
@@ -738,6 +738,10 @@ function dump($dn, $info)
 	 */
 	function serverPing($host, $port=389, $timeout=1)
 	{
+		// Replace ldaps:// by ssl://
+		if (preg_match('/^ldaps:\/\/([^\/]+)\/?$/',$host, $regs)) {
+			$host = 'ssl://'.$regs[1];
```
Comment (jfefe) [inline htdocs/core/class/ldap.class.php:743]: ssl:// ?

What about https://tools.ietf.org/html/rfc6176 and https://tools.ietf.org/html/rfc7568 ?

Comment (hregis) [reply htdocs/core/class/ldap.class.php:743]: @jfefe ;-)
http://php.net/manual/fr/function.fsockopen.php

Comment (jfefe) [reply htdocs/core/class/ldap.class.php:743]: Yep I saw that PHP have its own implementation of standards uri (see https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml) but  'ssl://' shouldn't be used anymore in favor of tls:// isn't it? The examples on the documentation are 15 years old :)

Comment (hregis) [reply htdocs/core/class/ldap.class.php:743]: @jfefe 
bah moi j'ai 45 ans et je fonctionne toujours pareil ! ;-)

**Pull request merged**